hosts/mno001/network.nix

```diff
@@ -1,34 +1,72 @@
 { pkgs, ... }:
 let
-  bond_name = "bond0";
+  bond_name = "bond";
 in
 {
 
-  # LACP on first two ports
-  networking.bonds."${bond_name}" = {
-    interfaces = [ "eno2" "eno3" ];
-    driverOptions = {
-      mode = "802.3ad";
-      lacp_rate = "fast";
-    };
+  networking = {
+    enableIPv6 = true;
+    useDHCP = false;
+
+    useNetworkd = true;
+    wireguard.enable = true;
   };
 
-  # Static IP Address
-  networking.interfaces."${bond_name}" = {
-    useDHCP = false;
-    ipv4.addresses = [
-      {
-        address = "212.111.245.178";
-        prefixLength = 29;
-      }
+  services.resolved = {
+    enable = true;
+    fallbackDns = [
+      "212.111.228.53" # IBH 1
+      "193.36.123.53" # IBH 2
+      "9.9.9.9" # QUAD 9
```

Comment:
Those resolvers are used in parallel which might not your intention. The IBH ns should be moved to resolved's [DNS](https://www.freedesktop.org/software/systemd/man/resolved.conf.html#DNS=) option and quad9 kept as [FallbackDNS](https://www.freedesktop.org/software/systemd/man/resolved.conf.html#FallbackDNS=).